### ISSUE TITLE
about azkaban one project create many flows

### ISSUE THREAD

**Issue by wallwind** [azkaban/azkaban#2319]
I have one question about azkaban can support one project create many flows ，how to do it>?

**Comment by djaiswal83** [comment]
Yes. you can have multiple flows in a given project. Let me know if this documentation helps,
https://azkaban.readthedocs.io/en/latest/createFlows.html
